e2e/test_operator_upgrades/operator_upgrades_test.go

```diff
@@ -453,7 +461,8 @@ var _ = Describe("Operator Upgrades", Label("e2e"), func() {
 				Skip("operator doesn't support feature for test case")
 			}
 
-			clusterSetup(beforeVersion)
+			// We ignore the availability check here since this check is sometimes flaky if not all coordinators are running.
+			clusterSetup(beforeVersion, false)
```

Comment:
Same here it should be fine to be enabled. I disabled the check in `upgrading a cluster and multiple processes are not restarted` as it could bring down the transaction system. Same in `upgrading a cluster and no coordinator is restarted`, this will cause the cluster to be unavailable for some time.

Reply:
Ok.